terraform/aws/addons/templates/keycloak.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: keycloak.org/v1alpha1
+kind: Keycloak
+metadata:
+  name: keycloak
+  namespace: ${namespace}
+  labels:
+    app: keycloak
+spec:
+  instances: 1
+  extensions:
+    - https://github.com/aerogear/keycloak-metrics-spi/releases/download/1.0.4/keycloak-metrics-spi-1.0.4.jar
```

Comment:
[p4] Maybe good to manage extension version from terraform

Reply:
passed template as is. not in scope